paper-toggle-button.html

```diff
@@ -59,120 +162,118 @@
     </div>
 
   </template>
+</dom-module>
```

Comment:
There isn't an advantage to pulling the script out of the `<dom-module>`, and it makes this patch much scarier than it is, so please don't do that :)